tests/main.js

```diff
@@ -2815,3 +2815,62 @@ testMany('None bundler emits esm with default nodeVersion', ['bundler_none'], as
 
   t.is(originalFile, bundledFile)
 })
+
+// The paths for the layer files are Unix only. Since we'll actually only use
+// layers in production (which is always Linux), we skip this test on Windows.
+if (platform !== 'win32') {
```

Comment:
So we don't want to/can use this in `netlify dev`?

Reply:
Hmm, good point. We probably will. I'll make it work on Windows. 👍🏻

Reply:
Done!